README.md

```diff
@@ -154,7 +152,6 @@ any given point in time, fully support a maximum of 2 LTS releases.
 [6.x]: https://nodejs.org/download/release/latest-v6.x/
 [8.x]: https://nodejs.org/download/release/latest-v8.x/
 [10.x]: https://nodejs.org/download/release/latest-v10.x/
-[Google Calendar]: https://calendar.google.com/calendar/ical/eln7trd6k7n6asgg49bu2vqn4s%40group.calendar.google.com/public/basic.ics
 [JSON]: schedule.json
 [lts module]: https://www.npmjs.com/package/lts
```

Comment:
It seems this ref can also be deleted.

Reply:
this is unrelated to the PR, would you be up for opening a separate PR to clean that up?

Reply:
Sorry, this is related because the link for this ref is deleted in the PR.

Reply:
ah, my apologies